### ISSUE TITLE
Center all text in header

### ISSUE THREAD

**Issue by liam4** [jay4/jay4.github.io#2]
As it is the separator, 'jay4' and 'chocolatepi' all have the same baseline. They should probably be centered.

**Comment by jay4** [comment]
7 hours ago was a long time ago.

Lots of things have changed since then. I took a shower. I ate dinner. I probably also ate lunch. I'm watching a movie now. I wasn't then. My dog wasn't out then. I took it out 2 times since then.

Wow. Look where we have come. In a time of future and prosperity. A time of dog walking. A time of lunch eating. A time of progress and innovation.

But somewhere, in the western spiral of the Milky Way Galaxy, on an insignificant yellow sun, a teenager created a website where another youth used his array of small colored boxes to file what is known as an "issue" on said website, sending random orders of bits to a device driver to type what has already been done, what has already been seen, and what has already been heard.

Little did he know that the one named "jay4" has already completed this and pushed to the repository just a few minutes ago.

tl;dr I already did this after you posted this :package:

**Comment by liam4** [comment]
Ha ha ha.. ;D
